dash/dash-renderer/src/APIController.react.js

```diff
@@ -46,6 +53,10 @@ const UnconnectedContainer = props => {
     if (!events.current) {
         events.current = new EventEmitter();
     }
+
+    const [libraryReady, setLibraryReady] = useState(false);
+    const onLibraryReady = useCallback(() => setLibraryReady(true), []);
```

Comment:
curious, what does `useCallback` do for us here? Prevents unnecessary rerender of `LibraryManager`?

Reply:
Yes, it doesn't need to be redefined every render, just the first.